### DISCUSSION TITLE
add button for close or hide the data summary

### DISCUSSION THREAD

**Discussion by romu545** [posit-dev/positron#4543] [Feedback]
![Captura de pantalla 2024-08-31 103508](https://github.com/user-attachments/assets/53945e3b-30ff-49e4-bb36-9cd61475cf2a)
Could you directly place a button to close or hide the data summary?
Just like I show in the image.

**Comment by jthomasmock** [comment]
Thanks for your feedback! We have a proposed UI for opening and closing: https://github.com/posit-dev/positron/issues/2823
